Microsoft.Azure.Cosmos/src/ChangeFeedProcessor/ChangeFeedProcessorContext.cs

```diff
@@ -23,5 +23,10 @@ public abstract class ChangeFeedProcessorContext
         /// Gets the headers related to the service response that provided the changes.
         /// </summary>
         public abstract Headers Headers { get; }
+
+        /// <summary>
+        /// Gets the feed range.
```

Comment:
No undocumentation please!

Reply:
What about:

```suggestion
        /// Gets the monitored container's <see cref="FeedRange"/> from which the changes come from.
```

Reply:
```suggestion
The <see cref="FeedRange"/> within the monitored container from which the changes emanated.
```
(mainly rewording to remove two 'from's in Matias's suggestion). (Or remove the first 'within' and end on 'arose within' ?)